src/components/input-field/input-field.scss

```diff
@@ -22,6 +22,19 @@
     display: none;
 }
 
+:host([type='textarea']) {
+    height: inherit; // This makes it possible for textarea in limel-form to stretch & span rows
+
+    .mdc-text-field.mdc-text-field--textarea {
+        height: var(--textarea-height, 100%);
+    }
+    textarea.mdc-text-field__input {
+        resize: vertical;
+        min-height: $height-of-mdc-text-field - 0.5rem;
+        max-height: 100%; // This will prevent the field to grow taller than its container, in grid forms for instance
```

Comment:
Nice.

This makes the resizing work as expected for "stand-alone" textareas, while the resizing becomes somewhat useless for textareas in forms…

I think that's good enough for this to be mergeable. We'll still have to take a look at other possibly expanding components, like limel-picker. We don't have an example for any of these in limel-form though (only a custom-picker, that isn't using `multiple` and therefore would need some tweaking), and I'm a little too tired to take on adding one right now.

@jgroth The layout stuff obviously isn't used yet, so if there are some things that don't yet work with that, I don't think that's the end of the world. But the one thing I'm a little concerned about is if the change to using a single column grid layout for the default layout might mean that limel-forms using the default layout also have the same problem with components that changes height?

Reply:
Yes, we should add a multi-line picker to one of the examples and play around with it. It's a bit hard for me to add fields to example forms, unless I can steal it from another example :D Can you guys do it plz…?

I don't think there is a **css-only** solution for fields that change size on the go. Probably some script should dynamically apply inline styles (for `grid-row` and `height`) directly on the element, to make the grid-cell span in rows (if user drags it to expand, or if it automatically expands). Such scrip should overwrite what's already set by the consumer in the schema.

#### one more thing
MD offers something that we may at some point use too, called "Multi-line fields". So it'd be nice if we're prepared for it.
![multi-line fields](https://user-images.githubusercontent.com/35954987/100760770-671d2e80-33f2-11eb-83c1-3a739f68066e.gif)

Anyhow, I don't think that the "default layout" has any problems with fields that increase in height. That default layout pretty much only cares about the gaps, at the moment.

Reply:
I've fixed the resizing issue for textareas 🙂